README.md

```diff
@@ -48,7 +48,7 @@ This repo is our playbook which contains our practices in software development a
 - [Document diagrams](/engineering/diagram.md)
 - [Writing REST API](/engineering/restful.md)
 - [Error convention](/engineering/error.md)
-- [Writing test & materials](/engineering/testing.md)
+- [Writing test & materials](/engineering/Testing/testing.md)
```

Comment:
There are several folders that keep the first letter in lowercase 🤔 I consider this **Testing** folder should be like that also

Reply:
renamed to lowercase as others